opba-protocols/xs2a-protocol/src/main/resources/processes/xs2a/consent/xs2a-authorize-consent.bpmn20.xml

```diff
@@ -110,7 +110,7 @@
         <omgdc:Bounds height="80.0" width="100.0" x="500.0" y="304.0"/>
```

Comment:
This file probably was changed accidentially

Reply:
Yes, it was rollbacked